app/views/buyers/accounts/_search_results.html.slim

```diff
@@ -52,7 +52,7 @@ table class="data" id="buyer_accounts"
           td class="plan" = h account.bought_account_plan.try(:name)
 
         - if can? :see, :multiple_applications
-          td = link_to (account.bought_cinstances.count), admin_buyers_account_applications_path(account)
+          td = link_to (current_user.accessible_cinstances.bought_by(account).count), admin_buyers_account_applications_path(account)
```

Comment:
Shouldn't we use https://github.com/3scale/porta/blob/92811d31658402cfd6f0e70c8a739965a5bd6a3a/app/controllers/buyers/applications_controller.rb#L50

Or this view is shared somewhere?

Reply:
I’d say that represents the whole collection of applications and the above are the applications within 1 developer account visible to the current provider user

Reply:
But this view is only used by index so I do not get it.